Remove all dead legacy code

jmauro opened the pull request:
Hello,

Since those template are not used anymore, we could remove those, don't you think?

cc @martinisoft or @onlyhavecans 

Regards,
JM

Commit:
Remove all dead legacy code

- Template file are not used anymore.

martinisoft commented on templates/default/exabgp.conf.erb:
```diff
@@ -1,10 +0,0 @@
-neighbor 127.0.0.1 {
```
This template is still required as a default: https://github.com/dnsimple/chef-exabgp/blob/5ab6b6fc03deeab874273ea99488fe890e7d569b/resources/exabgp.rb#L34

martinisoft replied:
The fact you removed it without a test breaking reveals a regression we should be covering in our test cases.

martinisoft reviewed (changes requested):
I can agree on the majority of these changes, but the template for the config needs to stay for sure. I'd like to make a failing test case for this too.

Commit:
Add the default cookbook used for the exabgp.conf's template

STATE:
Currently, there is no default cookbook source provided by the custom
resource.

FIX:
Add 'exabgp' as the default cookbook source.

jmauro commented:
Hello @martinisoft, I leave the default template alone and add a default cookbook source for the custom resource.

Let me know if this does the job

Regards,
JM
